src/whatsnew/2.3.rst

```diff
@@ -107,6 +107,43 @@ Upgrade Notes
 * :ghissue:`1680`: The embedded version of ``rebar`` used to build CouchDB has been
   updated to the last version of ``rebar2`` available. This assists in building on
   non-x86 platforms.
+* :ghissue:`1875`: Refuse building with known bad versions of Erlang.
```

Comment:
Is this supposed to be here? It's also duplicated below.

Reply:
Yeah, I want it both in the upgrade nodes and the per-version section.